data_structures/queue/priority_queue.py

```diff
@@ -0,0 +1,241 @@
+"""
+Pure Python implementation of Priority Queue using lists
+"""
```

Comment:
PEP8 encourages us to use specific exceptions instead of over using Exception. Please define your own OverflowException and UnderflowException
https://docs.python.org/3/tutorial/errors.html#user-defined-exceptions